examples/ex25.cpp

```diff
@@ -389,27 +389,22 @@ int main(int argc, char *argv[])
    //     applying any necessary transformations such as: assembly, eliminating
    //     boundary conditions, applying conforming constraints for
    //     non-conforming AMR, etc.
-   a.Assemble();
+   a.Assemble(0);
 
-   OperatorHandle Ah;
+   OperatorPtr A;
    Vector B, X;
-   a.FormLinearSystem(ess_tdof_list, x, b, Ah, X, B);
+   a.FormLinearSystem(ess_tdof_list, x, b, A, X, B);
 
-   // 13. Transform to monolithic SparseMatrix
-   SparseMatrix *A = Ah.As<ComplexSparseMatrix>()->GetSystemMatrix();
-
-   cout << "Size of linear system: " << A->Height() << endl;
```

Comment:
In the regression testing I noticed that this output was removed. Was this intentional?

I see a similar change the parallel version too.

Reply:
Yes it's intentional. The monolithic SparseMatrix is not needed anymore now